.github/workflows/release.yml

```
@@ -7,24 +7,24 @@ jobs:
   wheel-build:
     name: Build and Publish Release Artifacts
     runs-on: ubuntu-latest
+    environment: release
+    permissions:
+      id-token: write
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         name: Install Python
         with:
           python-version: '3.8'
       - name: Install Deps
-        run: pip install -U twine wheel
+        run: pip install -U wheel
       - name: Build Artifacts
         run: |
           python setup.py sdist
           python setup.py bdist_wheel
         shell: bash
       - uses: actions/upload-artifact@v3
         with:
-          path: ./dist/arraylias*
+          path: ./dist/qiskit*
```

I assume this was a copy and paste issue:

```suggestion
          path: ./dist/arraylias*
```